eclipse-collections-testutils/pom.xml

```diff
@@ -49,16 +49,15 @@
         </dependency>
 
         <!-- External dependencies -->
-
         <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-            <scope>compile</scope>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
         </dependency>
 
         <dependency>
-            <groupId>org.junit.jupiter</groupId>
-            <artifactId>junit-jupiter-api</artifactId>
+            <groupId>org.opentest4j</groupId>
+            <artifactId>opentest4j</artifactId>
+            <version>1.3.0</version>
```

Comment:
not crucial but consider moving the version number to the `dependencyManagement` section of the root pom.xml where we define the version of junit-jupiter-api.